README.rst

```diff
@@ -250,6 +250,25 @@ Requirements
   * ``contentstore.new_studio_mfe.use_new_export_page``: this feature flag will change the CMS to link to the new export page.
   * ``contentstore.new_studio_mfe.use_new_import_page``: this feature flag will change the CMS to link to the new import page.
 
+Feature: Tagging/Taxonomy Pages
+================================
+
+.. image:: ./docs/readme-images/feature-tagging-taxonomy-pages.png
+
+Requirements
+------------
+
+* ``edx-platform`` Waffle flags:
+
+  * ``contentstore.new_studio_mfe.use_tagging_taxonomy_list_page``: this feature flag must be enabled.
```

Comment:
nit:
```suggestion
  * ``new_studio_mfe.use_tagging_taxonomy_list_page``: this feature flag must be enabled.
```

Reply:
This is correct, but all the other feature flags use `contentstore.new_studio_mfe...` so that was a small mistake on our part. Doesn't really matter though.